packages/grid/_modules_/grid/models/params/gridVirtualPageChangeParams.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Object passed as parameter of the virtual page change event.
+ */
+export interface GridVirtualPageChangeParams {
+  /**
+   * The index of the current page.
+   */
+  currentPage: number;
+  /**
+   * The index of the next page.
+   */
+  nextPage: number;
```

Comment:
I don't think that we should expose these params. They are specific to the current virtualization implementation.

```suggestion
```